addons/base_vat/models/res_partner.py

```diff
@@ -382,6 +384,8 @@ def check_vat_nl(self, vat):
             return True
 
         vat = clean(vat, ' -.').upper().strip()
+        if not vat.startswith('NL'):
+            vat = 'NL' + vat
```

Comment:
FYI, the validation below always fails if there is no country code which is why I add it here.